android/src/main/java/com/brentvatne/exoplayer/FullScreenPlayerView.kt

```diff
@@ -127,4 +134,25 @@ class FullScreenPlayerView(
         layoutParams.setMargins(0, 0, 0, 0)
         return layoutParams
     }
+
+    private fun updateNavigationBarVisibility() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+            window?.setDecorFitsSystemWindows(!controlsConfig.hideNavigationBarOnFullScreenMode)
+            window?.insetsController?.let {
+                if (controlsConfig.hideNavigationBarOnFullScreenMode) {
+                    it.hide(WindowInsets.Type.navigationBars())
+                    it.systemBarsBehavior = WindowInsetsController.BEHAVIOR_SHOW_TRANSIENT_BARS_BY_SWIPE
+                } else {
+                    it.show(WindowInsets.Type.navigationBars())
+                }
+            }
+        } else {
+            @Suppress("DEPRECATION")
+            window?.decorView?.systemUiVisibility = if (controlsConfig.hideNavigationBarOnFullScreenMode) {
+                View.SYSTEM_UI_FLAG_HIDE_NAVIGATION
+            } else {
+                View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION
+            }
+        }
+    }
```

Comment:
I suggest using `WindowInsetsControllerCompat(window, window.decorView).hide(WindowInsetsCompat.Type.navigationBars())`

Reply:
Sorry for the delay in responding. I was busy the last few days. I applied your suggestion. Please check it out.